example/dj/apps/app/resource.py

```diff
@@ -137,3 +145,23 @@ class UserWithFormResource(BaseModelResource):
     read_obj_permission = True
     update_obj_permission = True
     delete_obj_permission = True
+
+
+class IssueForm(RESTModelForm):
+
+    created_by = SingleRelatedField('created_by')
+    leader = SingleRelatedField('leader')
+    another_users = MultipleRelatedField('watched_by', form_field=forms.ModelMultipleChoiceField(
```

Comment:
Souvísí tohle nějak s tímto PR? Nebo je to jen ukázka že je možné definovat vlastní pole ve formuláři?

Reply:
Je to tu za účelem testu a ano souvisí, Většinou dokumentaci píši společně s testy, takže vše co popisuji si otestuji